Source/Plugins/Core/com.equella.core/js/tsrc/modules/SearchModule.ts

```
@@ -221,18 +309,35 @@ export const searchItems = ({
   return OEQ.Search.search(API_BASE_URL, searchParams);
 };
 
+const getCollectionDetails = async (
+  collectionUuids: string[]
+): Promise<Collection[] | undefined> => {
+  const collectionList = await collectionListSummary([
+    OEQ.Acl.ACL_SEARCH_COLLECTION,
+  ]);
+  const filteredCollectionList = collectionList.filter((c) => {
+    return collectionUuids.includes(c.uuid);
+  });
+  return filteredCollectionList.length > 0
+    ? filteredCollectionList
+    : defaultSearchOptions.collections;
+};
+
+const getUserDetails = async (
+  userId: string
+): Promise<OEQ.UserQuery.UserDetails | undefined> => {
+  const userDetails = await resolveUsers([userId]);
+  return userDetails.length > 0 ? userDetails[0] : defaultSearchOptions.owner;
```

hmmm, if we can end up with more than 1 user, we should probably at least log a warning, but maybe we need to throw an error. Just assuming to use the first one is possibly a bit risky. :thinking: